politeiawww/www.go

```diff
@@ -393,7 +383,14 @@ func (p *politeiawww) handleChangePassword(w http.ResponseWriter, r *http.Reques
 	}
 	defer r.Body.Close()
 
-	reply, err := p.backend.ProcessChangePassword(email, cp)
+	user, err := p.getSessionUser(r)
+	if err != nil {
+		RespondWithError(w, r, 0,
+			"handleMe: getSessionUser %v", err)
```

Comment:
Should be handleChangePassword, not handleMe.

Reply:
fixed